### DISCUSSION TITLE
Speed reading widget

### DISCUSSION THREAD

**Discussion by fidel-perez** [SilentVoid13/Templater#377] [Templates Showcase]
I created a speedreading widget, I am pretty excited about it, hope someone else finds it useful. I will keep improving it / keep it updated with user comments:

https://gist.github.com/fidel-perez/a9b20e17f97bb37bfd0d2fbbdf3b7d6f

It is ugly but works perfectly:

![speedreadgif](https://user-images.githubusercontent.com/4046326/134499908-f125973f-4023-4df7-b45f-65cced57693b.gif)

**Comment by quetzlcoatlus** [comment]
This is sick thanks for submitting this!
